### ISSUE TITLE
Allow indicating auto-generated subtitles

### ISSUE THREAD

**Issue by cwillems** [openHPI/video-player#27]
Auto-generated (e.g. translated) subtitles should be indicated as such in the language dropdowns. Default indicator could be an asterisk (e.g zh*), overwriting the auto-subtitle style via CSS should be possible.

**Comment by cwillems** [comment]
We have a mockup for the preferred solution:
![player_mockup](https://user-images.githubusercontent.com/1996660/41031611-5a584902-6981-11e8-810e-f892293a073f.png)

**Comment by cwillems** [comment]
@soldag any plans when to tackle this?

**Comment by soldag** [comment]
I will have a look at this this week.

**Comment by soldag** [comment]
Done and included in [v2.0.1](https://github.com/openHPI/video-player/releases/tag/v2.0.1).

Captions are indicated as auto-translated if their type is set to `auto-generated`. Besides, an optional, user-friendly `name` can be specified, which is shown in the dropdown instead of the language code. For example, the state shown in the mockup can be achieved using the following config snippet: 
```json
"captions": [
  {
    "name": "English",
    "language": "en",
    "url": "captions_en.vtt"
  },
  {
    "name": "French",
    "language": "fr",
    "url": "captions_fr.vtt",
    "type": "auto-generated"
  },
  {
    "name": "German",
    "language": "de",
    "url": "captions_de.vtt",
    "type": "auto-generated"
  },
  {
    "name": "Spanish",
    "language": "es",
    "url": "captions_es.vtt",
    "type": "auto-generated"
  }
]
```
